package.json

```diff
@@ -0,0 +1,63 @@
+    "clean:data": "./mvnw spring-boot:run -DprojectCleanUp=true -DprojectSetUp=false -DchannelsImport=false -DcatalogImport=false -DreserveInStore=false -DordersImport=false -Dctp.projectKey=$npm_package_config_project -Dctp.clientId=$npm_package_config_client_id -Dctp.clientSecret=$npm_package_config_client_secret -Dctp.authUrl=$npm_package_config_authUrl -Dctp.apiUrl=$npm_package_config_apiUrl",
+    "import:projectSetup": "./mvnw spring-boot:run -DprojectCleanUp=false -DprojectSetUp=true -DchannelsImport=true -DcatalogImport=false -DreserveInStore=false -DordersImport=false -Dctp.projectKey=$npm_package_config_project -Dctp.clientId=$npm_package_config_client_id -Dctp.clientSecret=$npm_package_config_client_secret -Dctp.authUrl=$npm_package_config_authUrl -Dctp.apiUrl=$npm_package_config_apiUrl",
+    "import:catalogs": "./mvnw spring-boot:run -DprojectCleanUp=false -DprojectSetUp=false -DchannelsImport=false -DcatalogImport=true -DreserveInStore=false -DordersImport=false -Dctp.projectKey=$npm_package_config_project -Dctp.clientId=$npm_package_config_client_id -Dctp.clientSecret=$npm_package_config_client_secret -Dctp.authUrl=$npm_package_config_authUrl -Dctp.apiUrl=$npm_package_config_apiUrl",
+    "import:orders": "./mvnw spring-boot:run -DprojectCleanUp=false -DprojectSetUp=false -DcatalogImport=false -DreserveInStore=false -DordersImport=true -Dctp.projectKey=$npm_package_config_project -Dctp.clientId=$npm_package_config_client_id -Dctp.clientSecret=$npm_package_config_client_secret -Dctp.authUrl=$npm_package_config_authUrl -Dctp.apiUrl=$npm_package_config_apiUrl",
+    "import:zones": "npm run build && node dist/lib/zones.js --project $npm_package_config_project --client_id $npm_package_config_client_id --client_secret $npm_package_config_client_secret --apiUrl $npm_package_config_apiUrl --authUrl $npm_package_config_authUrl",
+    "import:shippingmethods": "npm run build && node dist/lib/shipping-method.js --project $npm_package_config_project --client_id $npm_package_config_client_id --client_secret $npm_package_config_client_secret --apiUrl $npm_package_config_apiUrl --authUrl $npm_package_config_authUrl",
+    "sync:products": "node_modules/.bin/product-csv-sync import --matchBy sku --csv data/products.csv --projectKey $npm_package_config_project --clientId $npm_package_config_client_id  --clientSecret $npm_package_config_client_secret --sphereProtocol https --sphereAuthProtocol https",
+    "sync:inventory": "node_modules/.bin/stock-import --logLevel debug --logSilent true --file data/inventory.csv --projectKey $npm_package_config_project --clientId $npm_package_config_client_id --clientSecret $npm_package_config_client_secret --sphereProtocol https --sphereAuthProtocol https",
```

Comment:
The documentation says "import:inventory" but the script is called "sync:inventory".   the difference does not make sense to me since some of the CLIs that actually sync (e.g. the categories and products) are called "import" and this one is "sync". 

It's not a bad idea to communicate which is actually syncing and which will fail if you run it a second time, but then it should match the actual behavior.

Reply:
I have the feeling that something is wrong with the data files here.  
Calling `sync:inventory` says  `'Stock entries to process: 26137'` but that should rather be something like 5000 if it does not include the store inventories (which should only be imported with `sync:inventory-stores`.

Reply:
the inventory.csv has 26137 lines and inventory-stores.csv has 130611 lines 
is that correct?

Reply:
Sorry, forget this comment.  I fell over the same thing like everybody else - the dataset has 2,968  *products* but these are 26k *skus* (=variants) since it's fashion with lots of size and color variations.